### DISCUSSION TITLE
Joystick device nodes in default dosemurc

### DISCUSSION THREAD

**Discussion by jharrison03** [dosemu2/dosemu2#2027] [General]
In the default dosemurc,  $_joy_device is using outdated nomenclature --

`# $_joy_device = "/dev/js0 /dev/js1`

It should probably be 

`# $_joy_device = "/dev/input/js0 /dev/input/js1`

**Comment by stsp** [comment]
Have you checked that it works?

**Reply by jharrison03** [reply]
I just checked this now, and I only have "/dev/input/js*", not "dev/js*".  

The "/dev/input/js*" works, not "dev/js*", so I think the config file should be changed.

**Reply by jharrison03** [reply]
I'll add this is a USB controller, not something on a traditional PC gameport, so I don't know if these are enumerated differently.

**Reply by skitt** [reply]
The joystick nodes moved to `/dev/input/jsX` quite a long time ago, both for traditional gameports and USB.
